src/services/attendees/attendee-router.ts

```diff
@@ -18,6 +20,26 @@ attendeeRouter.post("/", async (req, res, next) => {
     }
 });
 
+// generates a unique QR code for each attendee
+attendeeRouter.get("/qr/", RoleChecker([]), async (req, res, next) => {
+    try {
+        const userId = res.locals.payload.userId;
+        const expTime = Math.floor(Date.now() / 1000) + 20; // Current epoch time in seconds + 20 seconds
+        let hashStr = userId + "#" + expTime;
+        const iterations = 10;
+
+        for (let i = 0; i < iterations; i++) {
+            const hash = crypto.createHash("sha256");
+            hashStr = hash.update(hashStr).digest("hex");
+        }
+
+        const qrCodeString = `${hashStr}#${expTime}`;
```

Comment:
```suggestion
        const qrCodeString = `${hashStr}#${expTime}#${userId}`;
```
add in the userId here! we can explain during the meeting tomorrow